x-pack/plugins/fleet/common/types/rest_spec/epm.ts

```diff
@@ -46,6 +49,26 @@ export interface GetPackagesResponse {
   response?: PackageList;
 }
 
+interface InstalledPackage {
+  name: string;
+  version: string;
+  status: EpmPackageInstallStatus;
+  dataStreams: Array<{
+    name: string;
+    title: string;
+  }>;
+}
+export interface GetInstalledPackagesResponse {
+  items: InstalledPackage[];
+  total: number;
+  pageAfter?: SortResults;
```

Comment:
nit: we have used `searchAfter` for a similar param here: https://github.com/elastic/kibana/blob/069550d72a5549b0fe45f3290c0d856ea2bfd309/x-pack/plugins/fleet/server/services/agents/crud.ts#L212